features/events/syslog/src/test/java/org/opennms/netmgt/syslogd/SyslogdHandlerKafkaIT.java

```diff
@@ -28,24 +28,112 @@
 
 package org.opennms.netmgt.syslogd;
 
+import java.net.InetAddress;
 import java.util.Dictionary;
+import java.util.HashMap;
 import java.util.Map;
+import java.util.Properties;
 
+import kafka.server.KafkaConfig;
+import kafka.server.KafkaServer;
+
+import org.apache.camel.CamelContext;
+import org.apache.camel.Component;
+import org.apache.camel.Exchange;
+import org.apache.camel.Message;
+import org.apache.camel.Processor;
+import org.apache.camel.ProducerTemplate;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.kafka.KafkaComponent;
+import org.apache.camel.component.kafka.KafkaConstants;
+import org.apache.camel.impl.DefaultCamelContext;
+import org.apache.camel.impl.SimpleRegistry;
+import org.apache.camel.test.blueprint.CamelBlueprintTestSupport;
 import org.apache.camel.util.KeyValueHolder;
+import org.apache.curator.test.TestingServer;
+import org.junit.After;
+import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.opennms.core.test.OpenNMSJUnit4ClassRunner;
-import org.opennms.core.test.camel.CamelBlueprintTest;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.test.context.ContextConfiguration;
 
 @RunWith(OpenNMSJUnit4ClassRunner.class)
 @ContextConfiguration(locations = { "classpath:/META-INF/opennms/emptyContext.xml" })
-public class SyslogdHandlerKafkaIT extends CamelBlueprintTest {
```

Comment:
Change this class back so that it inherits from CamelBlueprintTest. Remove the unnecessary methods below (```doPreSetup()```, ```isUseAdviceWith()```, etc).